.github/workflows/ci.yml

```
@@ -136,7 +136,7 @@ jobs:
 
     strategy:
       matrix:
-        test-name: ['generic', 'express', 'quickstart', 'ap-flows', 'elements', 'sessions']
+        test-name: ['generic', 'express', 'quickstart', 'ap-flows', 'elements', 'sessions', 'expoWeb']
```

nit: just to align it with `ap-flows`
```suggestion
        test-name: ['generic', 'express', 'quickstart', 'ap-flows', 'elements', 'sessions', 'expo-web']
```